webapp/i18n/en.json

```diff
@@ -1951,7 +1951,7 @@
   "rename_channel.displayNameHolder": "Enter display name",
   "rename_channel.handleHolder": "lowercase alphanumeric characters",
   "rename_channel.lowercase": "Must be lowercase alphanumeric characters",
-  "rename_channel.maxLength": "This field must be less than 22 characters",
+  "rename_channel.maxLength": "This field must be less than {maxLength} characters",
```

Comment:
It's not particularly important here since this is a constant, but we should generally use `{maxLength, number}` for numbers so that they render 0 correctly. Without it, it treats 0 as an empty string since it's falsey.